packages/perseus-core/src/widgets/upgrade.ts

```diff
@@ -52,50 +51,7 @@ export const upgradeWidgetInfoToLatestVersion = (
         return oldWidgetInfo;
     }
 
-    // We do a clone here so that it's safe to mutate the input parameter
-    // in propUpgrades functions (which I will probably accidentally do at
-    // some point, and we would like to not break when that happens).
-    // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
-    let newEditorOptions = _.clone(oldWidgetInfo.options) || {};
-
-    const upgradePropsMap = getWidgetOptionsUpgrades(type);
-
-    // Empty props usually mean a newly created widget by the editor,
-    // and are always considerered up-to-date.
-    // Mostly, we'd rather not run upgrade functions on props that are
-    // not complete.
-    if (_.keys(newEditorOptions).length !== 0) {
-        // We loop through all the versions after the current version of
-        // the loaded widget, up to and including the latest version of the
-        // loaded widget, and run the upgrade function to bring our loaded
-        // widget's props up to that version.
-        // There is a little subtlety here in that we call
-        // upgradePropsMap[1] to upgrade *to* version 1,
-        // (not from version 1).
-        for (
-            let nextVersion = initialVersion.major + 1;
-            nextVersion <= latestVersion.major;
-            nextVersion++
-        ) {
-            if (upgradePropsMap[String(nextVersion)]) {
-                newEditorOptions =
-                    upgradePropsMap[String(nextVersion)](newEditorOptions);
-            } else {
-                throw new PerseusError(
-                    "No upgrade found for widget. Cannot render.",
-                    Errors.Internal,
-                    {
-                        metadata: {
-                            type,
-                            fromMajorVersion: nextVersion - 1,
-                            toMajorVersion: nextVersion,
-                            oldWidgetInfo: JSON.stringify(oldWidgetInfo),
-                        },
-                    },
-                );
-            }
-        }
-    }
+    let newEditorOptions = _.clone(oldWidgetInfo.options) ?? {};
```

Comment:
So as of this writing, I'm keeping `upgradeWidgetInfoToLatestVersion` because it seems to do other things besides purely upgrading widget options. I need to rename it.